### DISCUSSION TITLE
Highlighting and Ctrl+K to link should suggest internal page for linking

### DISCUSSION THREAD

**Discussion by kingfungdesign** [outline/outline#4949] [Ideas]
![msedge_un8nkaVXUo](https://user-images.githubusercontent.com/52369186/221667774-96f20e9a-a2e9-4957-830a-1ae37dadbec8.gif)

Currently, outline only suggests to create a new page with the highlighted text.
It should suggest to link to an internal page as if searching with the highlighted text. It feels so redundant to have to type the name again to find the page to link to.

**Comment by tommoor** [comment]
Good call, will add this very soon
